scholarx/archive/index.html

```diff
@@ -167,11 +148,25 @@ <h6>UNIVERSITY</h6>
                             <th>Details</th>
                         </tr>
                     </thead>
-                    <tbody id="team">
-                
+                    <tbody id="mentorProfiles">                    
                     </tbody>
                 </table>
             </div>
+            <!--mentees details-->
+            <div id="showMentees" class="mentees-table">
+                <table>
+                    <thead>
+                        <tr>
+                            <th></th>
```

Comment:
```suggestion
```

Reply:
Can you do these requested changes mentioned above? @janithlahirukariyawasam

Reply:
After that we can merge this

Reply:
> After that we can merge this
@YoshithaRathnayake Well I think this <th> tag is for aligning purpose. without this tag topic name will align with profile photo.

Reply:
@janithlahirukariyawasam can you remove the above table head tag

Reply:
> @janithlahirukariyawasam can you remove the above table head tag

You mean this line @anjula-sack ?
![image](https://user-images.githubusercontent.com/88109775/205477916-0569aac8-bf96-435b-a549-6ec0d3c06d34.png)

Actually, we have 4 columns, So If we remove that table head I think we have to face some aligning issue. 

![image](https://user-images.githubusercontent.com/88109775/205477940-801c15eb-e73b-4f9c-b8b8-77378582c48d.png)

**Name heading will be aligned with mentees profile picture**
![image](https://user-images.githubusercontent.com/88109775/205478058-69d486db-9e33-4c0a-8e0d-ab593bdccecc.png)

Reply:
oh yeah, no worries then